mistletoe/block_tokenizer.py

```diff
@@ -21,11 +20,11 @@ def __iter__(self):
     def __repr__(self):
         return repr(self.lines[self._index+1:])
 
-    def anchor(self):
-        self._anchor = self._index
+    def getpos(self):
+        return self._index
 
-    def reset(self):
-        self._index = self._anchor
+    def setpos(self, pos):
+        self._index = pos
```

Comment:
Can we please rename these to more standard `get_pos` and `set_pos`?

Or, I guess we could alternatively turn `pos` into [property](https://realpython.com/python-getter-setter/#using-properties-instead-of-getters-and-setters-the-python-way) - nice-to-have? :)

Reply:
Sure, I can rename to get_pos and set_pos if that's more standard. I think that's better than using a property, because the setter would kind-of have side effects.

Reply:
Thank you for the renaming.

> ... because the setter would kind-of have side effects

Out of curiosity, what side effects do you mean, besides storing the value?

Reply:
The side effect I was thinking of is that the next time you read a line, it will be from somewhere else in the buffer.

Reply:
Thanks for the clarification. So your point is probably that it might happen that some users of `lines.pos = X` could expect something else from this call than when using `lines.set_pos(X)`? I personally don't think so, but never mind, let's move on. :)